extension/src/XliffEditor/XliffEditorPanel.ts

```diff
@@ -278,6 +288,12 @@ function getNotesHtml(transunit: TransUnit): string {
             content += `${note.textContent.replace("-", html.br(2))}${html.br(2)}`;
         }
     });
+
+    if (transunit.targets.length > 1) {
+        transunit.targets.slice(1).forEach(trgt => {
+            content += `${trgt.translationToken} ${trgt.textContent}${html.br()}`;
+        });
+    }
```

Comment:
will they be shown first or last in the note? 
I think I would prefer if they was first...

Reply:
Definitely !